sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.projectVersion=0.0.1
 
 sonar.sources=src/app/, src/cases/, src/noc/, src/role-access, src/hearings
 sonar.test.inclusions=**/*.spec.ts
-sonar.exclusions=**/node_modules/**, **/*.html, **/*.scss, **/api/test/pact/**, **/*.module.ts, **/*.template.ts, , **/*.data.ts, src/app/app.routes.ts, src/role-access/containers/allocate-role/choose-duration/choose-duration.component.ts, src/role-access/store/effects/exclusion.effects.ts, src/app/components/hmcts-global-header/hmcts-global-header.component.ts, src/hearings/converters/judge-types.amended.converter.ts, src/hearings/converters/need-judge.amended.converter.ts, src/app/containers/app/app.component.ts, src/cases/containers/case-list/case-list.component.ts, src/cases/containers/case-search/case-search.component.ts, src/app/app.constants.ts, src/app/services/logger/appInsightsWrapper.ts, src/app/services/ccd-config/ccd-case.config.ts, src/app/store/effects/app.effects.ts, src/cases/resolvers/case-allocate-role-link-resolver.service.ts, src/hearings/containers/hearing-actuals/hearing-actuals-timing/hearing-actuals-timing.component.ts, src/cases/components/roles-and-access/roles-and-access.component.ts, src/hearings/guards/hearing-amendments-guard.ts, src/hearings/services/hearings-feature.service.ts, src/app/services/ccd-config/launch-darkly-defaults.constants.ts
+sonar.exclusions=**/node_modules/**, **/*.html, **/*.scss, **/api/test/pact/**, **/*.module.ts, **/*.template.ts, , **/*.data.ts, src/app/app.routes.ts, src/role-access/containers/allocate-role/choose-duration/choose-duration.component.ts, src/role-access/store/effects/exclusion.effects.ts, src/app/components/hmcts-global-header/hmcts-global-header.component.ts, src/hearings/converters/judge-types.amended.converter.ts, src/hearings/converters/need-judge.amended.converter.ts, src/app/containers/app/app.component.ts, src/cases/containers/case-list/case-list.component.ts, src/cases/containers/case-search/case-search.component.ts, src/app/app.constants.ts, src/app/services/logger/appInsightsWrapper.ts, src/app/services/ccd-config/ccd-case.config.ts, src/app/store/effects/app.effects.ts, src/cases/resolvers/case-allocate-role-link-resolver.service.ts, src/hearings/containers/hearing-actuals/hearing-actuals-timing/hearing-actuals-timing.component.ts, src/cases/components/roles-and-access/roles-and-access.component.ts, src/hearings/guards/hearing-amendments-guard.ts, src/hearings/services/hearings-feature.service.ts, src/app/services/ccd-config/launch-darkly-defaults.constants.ts, src/cases/case-feature.routes.ts
```

Comment:
Could add all routes files here maybe? app.routes.ts already referenced here for example

Reply:
@connorpgpmcelroy sonar was complaining about only case-feature.routes.ts for duplication of code.

Reply:
I know but might be worth adding all routes.ts files so this doesn't get flagged again

Reply:
Maybe could add tech debt ticket to look into this and the sonar exclusions further?

Reply:
@connorpgpmcelroy I have considered all routes file